pgmpy/models/BayesianModel.py

```diff
@@ -315,10 +315,12 @@ def remove_cpds(self, *cpds):
                 cpd = self.get_cpds(cpd)
             self.cpds.remove(cpd)
 
-    def get_cardinality(self, node):
+    def get_cardinality(self, node=None):
```

Comment:
@raghavg7796 Then I think it should just return a dictionary without taking any argument.